ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main__tempo-product-ocp-14-lp-interop.yaml

```diff
@@ -0,0 +1,81 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  operator-sdk:
+    name: "4.14"
+    namespace: origin
+    tag: operator-sdk
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+canonical_go_repository: github.com/grafana/tempo-operator
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: tempo-operator
+- context_dir: .
+  dockerfile_path: tests/Dockerfile
+  to: tempo-tests-runner
+operator:
+  bundles:
+  - as: tempo-bundle
+    context_dir: ./bundle/openshift
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: ghcr.io/grafana/tempo-operator/tempo-operator:v0.2.0
+    with: pipeline:tempo-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: tempo-interop-aws
+  cron: 00 21 * * */3
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
```

Comment:
Why the base domain and EO_ env vars are set?

Reply:
Since this job is for the interop, it is using the cluster profile provided by Cloud Services and Platform Integration team, the cluster profiles requires setting the domain. The domain will differ when we use different profiles. The EO_ vars are from the elasticsearch operator install step, they specify the elasticsearch operator branch and operator source to use for installation.

Reply:
> , it is using the cluster profile provided by Cloud Services and Platform Integration team,

How is this cluster profile different to the "default"? 

> The EO_ vars are from the elasticsearch operator install step, they specify the elasticsearch operator branch and operator source to use for installation.

Maybe we could put some comments here to document it?

Reply:
The CSPI profile is same as other IPI profiles, the CSPI team requested us to add the interop jobs with the profile. As for the steps, we will be adding documentation to all the steps and jobs.